metadata-ingestion/src/datahub/ingestion/graph/client.py

```diff
@@ -1736,6 +1737,89 @@ def get_entities_v2(
                     retval[entity_urn][aspect_key] = aspect_value
         return retval
 
+    def get_entities_v3(
+        self,
+        entity_name: str,
+        urns: List[str],
+        aspects: Optional[List[str]] = None,
+    ) -> Dict[str, Dict[str, _Aspect]]:
```

Comment:
`/openapi/v3/entity/*/batchGet` endpoints have a _with_system_metadata_ flag.

Adding the flag is easy but the response model becomes much more complex, sort of: 

```
Dict[str, Dict[str, Tuple[_Aspect, Optional[<system_metadata>]] ]]
```

So just skipped it, for the moment.

Reply:
that's fine for now - eventually we may have a use-case for it, and we can add it then (and use typing.overload to make the types work)

in another place in the sdk, we made it so that when you request with system metadata we return MCPWs - maybe that could work down the line here too?